src/yunohost/app.py

```diff
@@ -2068,11 +2068,15 @@ def _parse_action_args_in_yunohost_format(args, action_args, auth=None):
 
                 # Append extra strings
                 if arg_type == 'boolean':
-                    ask_string += ' [0 | 1]'
+                    ask_string += ' [yes | no]'
                 elif arg_choices:
                     ask_string += ' [{0}]'.format(' | '.join(arg_choices))
+
                 if arg_default is not None:
-                    ask_string += ' (default: {0})'.format(arg_default)
+                    if arg_type == 'boolean':
+                        ask_string += ' (default: {0})'.format("yes" if arg_type == 1 else "no")
```

Comment:
Uuh, here we just checked that `arg_type == 'boolean'` and now we check if `arg_type == 1` ? Am I missing something ?

Reply:
Ah no, should be arg_defaut, my bad.